### ISSUE TITLE
Events Not Merging

### ISSUE THREAD

**Issue by ilyg** [imightbeamy/gcal-multical-event-merge#76]
The tool is not merging the events that are similar/the same. When I toggle on or off no change happens even with those events which are supposed to merge. 

For instance, my schedule shift is both on the attendance calendar and I have "invited" myself and so it should merge showing that it is, in essence, the same event. 

This extension has worked for me outside of corporate and I'm sad that it's not working for me here! 

Your aid is greatly appreciated. 

<img width="249" alt="Screen Shot 2019-06-17 at 9 04 49 AM" src="https://user-images.githubusercontent.com/51922677/59619273-03f4ea80-90df-11e9-8190-0d87bdad24a8.png">

**Comment by gregschwartz** [comment]
Seeing same issue: 
<img width="398" alt="Screen Shot 2019-06-21 at 10 52 52 AM" src="https://user-images.githubusercontent.com/206568/59941868-248caf80-9413-11e9-93ed-38352f8b4dcf.png">

I even tried copying the green event to the blue calendar just now, to make sure the events are the same.

**Comment by ilyg** [comment]
Yes! I make sure to just hit duplicate or to "invite" another calendar but
it won't merge at all.

Thanks for submitting this screenshot!

I L A R I A  G H A T T A S
REWS MTV | ghattas@google.com
[wo]MANPOWER, onsite at Google

*Work is love made visible *//* Khalil Gibran*

**Comment by imightbeamy** [comment]
I've been trying to replicate what you're both seeing, but haven't been able too :(

Here are some questions that might help me get to the bottom of it:
* If you copy those events to another week, do they merge?
* Are you seeing other merged events, or are no events merging for you at all? 
* Can you open the javascript console and screenshot anything you see there? 
    * You can do that in Chrome by going to "View" => "Developer" => "JavaScript Console".

**Comment by ilyg** [comment]
I'd be happy to get on a GVC with you so we can troubleshoot this in real
time! Let me know if you'd like to do that.

- none of my events are merging at all :(

I L A R I A  G H A T T A S
REWS MTV | ghattas@google.com
[wo]MANPOWER, onsite at Google

*Work is love made visible *//* Khalil Gibran*

**Comment by gregschwartz** [comment]
No events are merging at all, on many different weeks, even after a fresh restart. JS console has something interesting!

````
events.user.js:157   Event merge is disabled
````

That code is:
````
setTimeout(() => chrome.storage.local.get('disabled', storage => {
  console.log(`Event merge is ${storage.disabled ? 'disabled' : 'enabled'}`);
  if (!storage.disabled) {
    const observer = new MutationObserver(init);
    observer.observe(document.querySelector('body'), { childList: true, subtree: true, attributes: true });
  }

  chrome.storage.onChanged.addListener(() => window.location.reload())
}), 10);
``````

No clue why local storage would be disabled, here's the settings for gCal:
![Screen Shot 2019-07-01 at 7 57 06 PM](https://user-images.githubusercontent.com/206568/60479391-841c6380-9c3a-11e9-96cc-bb0900f83f59.png)
![Screen Shot 2019-07-01 at 7 57 36 PM](https://user-images.githubusercontent.com/206568/60479397-87175400-9c3a-11e9-88a8-beb16b5a71e4.png)

**Comment by cyberbit** [comment]
Same issue here, this is a screenshot of my console:
![image](https://user-images.githubusercontent.com/7350183/60814725-c4f50a80-a15b-11e9-9ba2-96a6f536fdf9.png)

**Comment by Xaxetrov** [comment]
Events are merging in month view but not in some week views.
